### PR TITLE
feat: add readme

### PR DESCRIPTION
**Pull request by karlknowit**
## Explain the changes you’ve made
Add missing README.md file to version-api

## Explain why these changes are made
The README was missing

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1vkr59k Skapa README.md fil till version-api](https://app.clickup.com/t/1vkr59k)

File: services/version-api/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+A running instance of an API GATEWAY on AWS that includes a gateway resource named /version. You can find and deploy this in our [resource](https://github.com/helsingborg-stad/helsingborg-io-sls-resources/tree/dev/services/gateway/resources/version) repository.
+
+### AWS PARAMETERSTORE (REQUIRED)
+
+A setup of `versionEnvs` AWS parameterstore on aws. This can be created from the `resource` api. You can find and deploy this in our [resource](https://github.com/helsingborg-stad/helsingborg-io-sls-resources/tree/dev/services/parameterStore/envs) repository.
+
+### Installation
+
+```bash
+$ npm install
```
Comment (dannil76) [inline services/version-api/README.md:28]: Replace with `yarn`:)

File: services/version-api/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+When you tear down the service, serverless will remove all resources created in the AWS console, including emptying the `deploymentBucket`.
+
+### Testing (Jest)
+
+Run test command:
+
+```bash
+$ npm run test
```
Comment (dannil76) [inline services/version-api/README.md:56]: Replace with `yarn test`

File: services/version-api/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+Run test command with [options](https://jestjs.io/docs/cli#options):
+
+```bash
+$ npm run test --[option_1] --[option_2]
```
Comment (dannil76) [inline services/version-api/README.md:62]: Replace with `yarn test --[option_1] --[option_2]`:)

File: services/version-api/README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+#### Expected Response
+
+```
+{
+    "jsonapi": {
+        "version": "1.0"
+    },
+    "data": {
+        "status": "OK | UPDATE_OPTIONAL | UPDATE_REQUIRED",
+        "updateUrl": "URL",
```
Comment (dannil76) [inline services/version-api/README.md:90]: Update to reflect implementation in code :)
`
"type": "getStatus",
"attributes": {
   "status": ...
   "updateUrl": ...
}
`

**Review by dannil76 — CHANGES_REQUESTED** [review]

**Review by dannil76 — APPROVED** [review]

**Pull request merged**